js/src/analysis/classicEditorData.js

```diff
@@ -100,18 +103,25 @@ class ClassicEditorData {
 	subscribeToElements() {
 		this.subscribeToInputElement( "title", "title" );
 		this.subscribeToInputElement( "excerpt", "excerpt" );
+		this.subscribeToInputElement( "excerpt", "excerpt", false );
 		this.subscribeToInputElement( "excerpt", "excerpt_only" );
+		let cb = () => {
+			this._store.dispatch( setDocumentText( this.getContent() ) );
+			mapDocumentToDisplayData( this._store );
+		};
+		tmceHelper.addEventHandler( tmceId, [ "input", "change", "cut", "paste" ], cb );
 	}
 
 	/**
 	 * Subscribes to an element via its id, and sets a callback.
 	 *
-	 * @param {string} elementId          The id of the element to subscribe to.
-	 * @param {string} targetReplaceVar   The name of the replacevar the value should be sent to.
+	 * @param {string}  elementId            The id of the element to subscribe to.
+	 * @param {string}  targetField          The name of the field the value should be sent to.
+	 * @param {boolean} forReplaceVariable   Whether this is a subscription for the replaceVariables.
```

Comment:
There should be just one space before `Whether` and the previous lines should be adjusted accordingly